### PR TITLE
LPS-174834 Add priority to listTypeDefinitionId if exists in objectField when posting object definition

### PR DESCRIPTION
**Pull request by murilostodolni**
Related: https://issues.liferay.com/browse/LPS-174834

### CODE REVIEW

**Comment by joanagrego** [comment]
Just starting reviewing :)

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldUtil.java [hunk 1]
```diff
@@ -63,6 +63,10 @@ public static long addListTypeDefinition(
 			return 0;
 		}
 
+		if (Validator.isNotNull(objectField.getListTypeDefinitionId())) {
+			return objectField.getListTypeDefinitionId();
+		}
```
Comment (carolmariaabb) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldUtil.java:68]: @murilostodolni do we need to check if the `objectField.getListTypeDefinitionId()` is valid?

Comment (murilostodolni) [reply modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldUtil.java:68]: Hi @carolmariaabb, thanks for the question. The validation if listTypeDefinition exist with this ID is made in service.

**Comment by brianchandotcom** [comment]
@carolmariaabb @murilostodolni I made it e4cb10c38d3df50877d83a2b223e3a55e20097e0 instead

**Comment by murilostodolni** [comment]
ci:close

**Pull request closed without merging**